src/components/header/header.jsx

```diff
@@ -48,7 +48,7 @@ const Header = () => {
           </div>
           <div className="navbarIconContainer  d-flex justify-content-center align-items-center">
             <a href="ss">
-              <ProfilePic className="profile d-flex justify-content-center align-items-center" />
+              {/* <ProfilePic className="profile d-flex justify-content-center align-items-center" /> */}
```

Comment:
@abdulmaleekadams Can you check on your svg for this profile pic? it is the cause for the blue thing on top right of the page

Reply:
@mrxshinji on it

Reply:
@abdulmaleekadams when you free, review this pr first and approve if its ok. Will merge this first before other pr to create conflict and slowly solve from there i guess

Reply:
@mrxshinji @abdulmaleekadams what's the status of this PR?

Reply:
I think this is resolved in [UPDATE PAGE FIX](https://github.com/appwex/react-ui-components/pull/66), @mrxshinji please confirm it.

Reply:
i have other fix as well, due to some merge conflict wasnt solve properly..

Reply:
Okay

Reply:
@aftabrehan  I have fixed it

Reply:
ok I will check